### PR TITLE
Add Czech translations + support multiple plurals

### PR DESCRIPTION
**Pull request by danielhollas**
Hi there!

First of all, thanks so much for making this theme. It's minimalistic yet has everything I need! (math support, dark mode)

I made Czech translations and also a small modification to allow for multiple plurals in i18n translations (in some langs, the noun for depends on the number before it).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Fuji is a minimal Hugo theme with full dark mode support and GitHub Primer markd
 
 After the release of v2, **there may be major changes in the image lazyload shortcode** to make it easier to use. If you want to change the color scheme, please check [🔧 Advanced configuration](#-advanced-configuration).
 
-Now supported i18n langs: en, zh-hans, zh-hant, ja, nl, pt-pt. Check the i18n folder to add more languages.
+Supported languages: `cs`, `en`, `ja`, `nl`, `pt-pt`, `zh-hans`, `zh-hant`. Check the i18n folder to add more languages.
```
Comment (danielhollas) [inline README.md:11]: I reordered them alphabetically and added czech.

File: i18n/cs.toml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+[sidebarToc]
+    other = "Obsah"
+
+[sidebarPages]
+    other = "Navigace"
```
Comment (danielhollas) [inline i18n/cs.toml:5]: I translated "Pages" as "Navigation", seemed to me as a better description.

**Comment by dsrkafuu** [comment]
Thanks for your contribution! 😉
Regarding the multiple plurals, perhaps the Portuguese(pt-pt) should also apply. But since it was translated by others and I don't know anything about the language, I will just keep that for the time being.

**Comment by danielhollas** [comment]
Thanks for merging! Yeah, it is relevant for other langs (at the very least the "one" form), but I didn't want to mess with them.

**Pull request merged**